code/modules/research/bepis.dm

```diff
@@ -178,19 +178,44 @@
 /obj/machinery/rnd/bepis/ui_data(mob/user)
 	var/list/data = list()
 	var/powered = FALSE
+	var/zvalue = (banked_cash - (major_threshold - positive_cash_offset - negative_cash_offset))/(std)
+	var/std_success = 0
+	var/prob_success = 0
+	//Admittedly this is messy, but not nearly as messy as the alternative, which is jury-rigging an entire Z-table into the code, or making an adaptive z-table.
```

Comment:
Calculations were doable in javascript in my opinion, if reasoning was "DM slow and ugly".

Reply:
If there's a better way to do so, I'm afraid it's out of my skill level I'm afraid. Still, this is certainly better than going in blind when your chance of success is <1%